CONTRIBUTING.md

```diff
@@ -114,6 +114,13 @@ If you use Mac, you may check follow path while looking for SDK:
 Now you can use the run configurations provided by the plugin source code to
 run and play.
 
+There are multiple branches of the plugin which correspond to different versions
+of the IntelliJ Platform:
+
+- 141 -> can be used for IDEA 14
```

Comment:
Updated. If it's ok for you, I'll commit the updates to the versions as the status changes directly without going via PR next time. Thank you.